src/View/CsvView.php

```diff
@@ -442,6 +454,23 @@ protected function _generateRow($row = null)
             }
         }
 
+        //bom must be added after encoding
+        if ($this->viewVars['_bom'] === true) {
```

Comment:
Strictness here should be a BC break for some, better to stick to the current trueish check IMO.